server/pension/tests.py

```diff
@@ -1,3 +1,7 @@
 from django.test import TestCase
 
-# Create your tests here.
+class TestPension(TestCase):
+
+    def test_index_view(self):
+        response = self.client.get('/api/')
```

Comment:
@ranmoshe 
maybe we should add another similar test to check without the last `/` ?

```PY
response = self.client.get('/api')
self.assertEqual(response.status_code, 200)
```